modular_skyrat/master_files/code/modules/roundstart/lawsets.dm

```diff
@@ -0,0 +1,34 @@
+/datum/station_trait/unique_ai
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. To make your life miserable, we have removed your additional lawset boards. Good luck!"
+// Don't roll the default lawsets, please!
+/pick_weighted_lawset()
```

Comment:
what is this

Reply:
horrifying abuse of byond

Reply:
override on a root level proc

Reply:
oh you mean the upper part

ok so people keep referring to the build in "Don't change the laws" part is policy when its not, and piss their metaphorical pants about it to the misery of everyone else in the round, as command mains that are also human mains will use it to clutch their pearls at the suggestion it should be changed.

Reply:
There's no point in this trait existing if you're going to remove the blocker on resetting the laws.

Reply:
> There's no point in this trait existing if you're going to remove the blocker on resetting the laws.

Not removing the blocker at all, just the suggestion that admins demand the game be played in a certain way

Reply:
i see, carry on